_pages/testing.md

```diff
@@ -1,21 +1,22 @@
 ---
 title: Testing your app
 lead: >
-  Once you’ve created your app and implemented an identity protocol, you can now register it in our dashboard and start testing.
+  Once you’ve created your app and implemented an identity protocol, you can now register it in the test environment dashboard and start testing.
 redirect_from:
   - /registering-your-sp/
   - /register/
 ---
 
 # Using the dashboard
 
-The login.gov dashboard is the place you'll manage your team's test applications.
+Login.gov provides an open sandbox enviroment to create and test integrations between login.gov and you applications. In the sandbox environment, we provide a dashboard where you can manage your test applications. While the dashboard is open to use, you must request access.
 
-1. First, the login.gov team will create accounts for you and your team members. Once you have an account, head over to the dashboard at [dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov)
-2. In the upper-right corner, click **Log in**. You'll be prompted to sign in or create an account with the login.gov IdP in the agency integration environment (hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov)).
+1. Request that accounts your emails get added to the dashboard application in the test sandbox environment. You can request [access here](https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e). The login.gov team will create accounts for you and your team members. Once you have an account, head over to the dashboard at [dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov).
```

Comment:
We'll keep this for now, but will likely change the way developers request sandbox access via email

Reply:
That makes sense